src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75d;
+    private static final int GROW_NUMBER = 2;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
+        int index = getIndex(key);
+        if (table[index] == null) {
+            table[index] = new Node<>(key, value, null);
```

Comment:
```suggestion
            table[index] = new Node<>(key, value);
```
create consructor with 2 parameters instead, the 3d parameter is always null in this solution